inductiva/resources/machine_cluster.py

```diff
@@ -52,7 +53,8 @@ def __init__(self,
     @classmethod
     def from_api_response(cls, resp: dict):
         machine_group = super().from_api_response(resp)
-        machine_group.num_machines = int(resp["num_vms"])
+        machine_group.num_machines = int(resp["max_vms"])
```

Comment:
Previously, num_machines always had the number of machines requested, this means that to keep this variable with the same number, number_machines now has max_vms.
I did this in hopes of not breaking any code that was counting on num_machines having the maximum number of machines.

Reply:
please confirm with @begaiym-k this change to be sure nothing is breaking

Reply:
> please confirm with @begaiym-k this change to be sure nothing is breaking

I had it wrong. num_vms never changed. It always had the current active machines. I will have num_machines for the active machines and _requested_machines for the max_vms